contrib/ros/worlds/experiment_configuration.py

```diff
@@ -0,0 +1,3 @@
+VICON_TOPICS = []
```

Comment:
use the pre-existing `garage/config_personal.py` (and `garage/config_personal_template.py`)